readthedocs/doc_builder/python_environments.py

```diff
@@ -89,20 +89,17 @@ def install_package(self, install):
                 bin_path=self.venv_bin(),
             )
 
-    def venv_bin(self, filename=None, conda=False):
+    def venv_bin(self, prefixes, filename=None):
```

Comment:
Do we even need this function? It seems like it's just a wrapper around `os.path.join`?

Reply:
We don't _need it_, but it's used a lot of times in this file. I guess it was created to avoid repeating these lines that many times only since this function it's not super smart 😄 .

I counted them. It's used 19 times.